examples/api/lib/widgets/inherited_model/inherited_model.0.dart

```diff
@@ -0,0 +1,165 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for InheritedModel
+
+import 'package:flutter/material.dart';
+
+enum LogoAspect { backgroundColor, large }
```

Comment:
nit: make this private: `_LogoAspect`

Reply:
`updateShouldNotifyDependent` has a lint warning when using  private types.

> `Avoid using private types in public APIs.`